projects/igniteui-angular/schematics/ng-add/index.spec.ts

```diff
@@ -90,7 +90,8 @@ describe('ng-add schematics', () => {
     await runner.runSchematic('ng-add', { normalizeCss: false }, tree);
     const pkgJsonData = JSON.parse(tree.readContent('/package.json'));
     expect(pkgJsonData.dependencies['fflate']).toBeTruthy();
-    expect(pkgJsonData.dependencies['hammerjs']).toBeTruthy();
+    // hammer is optional now.
+    //expect(pkgJsonData.dependencies['hammerjs']).toBeTruthy();
```

Comment:
Can we make this check for optional deps?

Reply:
I don't think ng-add adds the optional dependencies by default. I could take a look and see if it can be configured to add them in some way for this test in particular, but out of the box it would not.

Reply:
> I don't think ng-add adds the optional dependencies by default. I could take a look and see if it can be configured to add them in some way for this test in particular, but out of the box it would not.

That is true. Ignore my comment and just fix the conflict :)